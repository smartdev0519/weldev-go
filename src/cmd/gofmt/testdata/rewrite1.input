package main

type Foo int

func main() {
	var a Foo
	println(a)
}
