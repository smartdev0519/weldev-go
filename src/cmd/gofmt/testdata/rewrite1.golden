package main

type Bar int

func main() {
	var a Bar
	println(a)
}
